docs/dev/analysis/features/text/pictures-in-runs.rst

```diff
@@ -0,0 +1,105 @@
+Shapes in Runs
+==============
+
+This document is aimed at accessing image information when reading docx files.  Adding image information as part of creating docx files is handled elsewhere.
+
+A Run <w:r> can contain a Drawing <w:drawing>, which can can contain an Inline <wp:inline>, which can contain a Graphic <a:graphic>, which can contain graphicData <a:graphicData>, which can contain a Picture <pic:pic>.  A Picture can contain a blipFill <pic:blipFill> and a "Shape Properties" spPr <pic:spPr>.  A blipFill contains a Blip <a:blip> with an embed property that provides the graphic's Resource ID (rId) needed to look the image up in document.part.rels.  The spPr contains informtion about the image's size.  All of this except the Drawing element are documented in docs/dev/analysis/features/shapes/shapes-inline.rst.
+
+
+Word Behavior
+-------------
+
+In theory, a Run can contain multiple Drawings.  In practice, I have not found a way to create a Run with more than a single Drawing.
+
+
+Protocol
+--------
+
+.. highlight:: python
+
+To stay parallel with document.add_picture() behavior, which creates an Inline object, the Run should have an Inlines property, producing a list of Inline objects.
```

Comment:
Be careful about mixing API object names and XML element names. The collection type here is `InlineShapes` and should already exist because its available from `Document` on `Document.inline_shapes`. I think you'll find the MS API property is `Document.InlineShapes`: https://msdn.microsoft.com/en-us/library/office/ff844996.aspx